cache-config/t3c-apply/config/config.go

```diff
@@ -580,6 +618,13 @@ If any of the related flags are also set, they override the mode's default behav
 		return Cfg{}, errors.New("Initializing loggers: " + err.Error() + "\n")
 	}
 
+	if len(fatalLogStrs) > 0 {
+		for _, str := range fatalLogStrs {
+			str = strings.TrimSpace(str)
+			log.Warnln(str)
```

Comment:
shouldn't fatal errors be logged to the error stream rather than the warning stream?

Reply:
updated to `log.Errorln(str)`